components/OgImage/EventShareInstagram.vue

```
@@ -1,148 +1,95 @@
-  <div
-    style="
-      position: relative;
-      background-color: #fce6e8;
-      color: #25282b;
-      width: 100%;
-      display: flex;
-      flex-direction: column;
-      align-items: center;
-      height: 100%;
-    "
+  <div 
```

ESLint

[error] Delete `·`

(prettier/prettier)